### PR TITLE
Fix/3241  add a more specific error message and update translations

### PR DESCRIPTION
**Pull request by gbonaspetti**
#3241 

## Checkpoints

The goal of this section is to help code reviewers and the merge request author to do the required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present
- [x] Automated tests have been written (covering the feature or non regression), or *at least* an issue has been created to implement the test

**For developers**

- [x] Manual tests done to ensure stability on whole application layers, issue expectation follow
- [x] The original issue has been updated with a short summary of the implemented solution

**For the quality team**

- [x] Tested by the quality team

### CODE REVIEW

File: frontend_lib/i18next.scanner/fr/translation.json [hunk 1]
```diff
@@ -83,5 +90,7 @@
   "User + create shared spaces, add members in shared spaces": "Utilisateur + créer des espaces partagés, ajouter des membres dans les espaces partagés",
   "Shared space manager": "Responsable",
   "Refresh?": "Rafraîchir ?",
-  "If you refresh, you will lose the current changes": "Si vous vous rafraîchissez, vous perdrez les modifications actuelles"
+  "If you refresh, you will lose the current changes": "Si vous vous rafraîchissez, vous perdrez les modifications actuelles",
```
Comment (raphj) [inline frontend_lib/i18next.scanner/fr/translation.json:93]:
```suggestion
  "If you refresh, you will lose the current changes": "Si vous rafraîchissez, vous perdrez les modifications actuelles",
```

Suggestion: If you refresh the page

File: frontend_lib/i18next.scanner/fr/translation.json [hunk 2]
```diff
@@ -33,6 +34,12 @@
   "Search user...": "Rechercher un utilisateur...",
   "Unknown error, please inform an administrator": "Erreur inconnue, veuillez contacter un administrateur",
   "Error detail": "Détail de l'erreur",
+  "Error while archiving content": "Erreur lors de l'archivage d'un contenu",
+  "Error while changing status": "Erreur lors d'un changement de statut",
+  "Error while deleting content": "Erreur lors de la suppression d'un contenu",
+  "Error while restoring content": "Erreur lors de la restauration du contenu",
+  "Error while saving new comment": "Erreur lors de l'enregistrement d'un nouveau commentaire",
+  "Error while saving new title": "Erreur lors de l'enregistrement d'un nouveau titre",
```
Comment (raphj) [inline frontend_lib/i18next.scanner/fr/translation.json:42]:
```suggestion
  "Error while saving new title": "Erreur lors de l'enregistrement du nouveau titre",
```

"Error while changing the title?"

**Comment by Skylsmoi** [comment]
@gbonaspetti This PR is lacking things to allow reviewers to check the checkbox about tests

**Comment by gbonaspetti** [comment]
> @gbonaspetti This PR is lacking things to allow reviewers to check the checkbox about tests

Sorry, I forgot to justify. Functional tests were not written because the error case for which the message was added is done using TLMs and at the moment we don't have how to simulate two windows/tabs in the tests. The other cases which I added the translations are equally difficult to simulate because they are default cases, usually caused by a 500 error in the backend.

**Pull request merged**